src/containers/Login.vue

```diff
@@ -233,7 +233,6 @@ export default Vue.extend({
         .then(doc => {
           if (doc.exists) {
             this.performingRequest = false;
-            this.$store.commit('setCurrentUser', user);
```

Comment:
removed because it's never read before this commit, and now it's auto set when the user is logged in